source/_posts/2020-07-11-android-18-110-release.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "Home Assistant Companion for Android 1.8 - 1.10"
+description: "Today we're releasing a big update for our Android companion app including lots of improvements to notifications, biometrics support and a new geocoded sensor."
+date: 2020-06-18 00:00:00
+date_formatted: "June 18, 2020"
+author: Daniel Shokouhi
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-07-android-18-110-release/biometric_unlock.png
+---
+
+There have been lots of more improvements to the android app since we last had a release post so lets go over some of the highlights!
```

Comment:
```suggestion
There have been lots of more improvements to the Android app since we last had a release post so let's go over some of the highlights!
```